### PR TITLE
osd/OSDMap: pg_to_osds -> pg_to_raw_osds, fix libcephfs caller to get acting osds

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by gregsfortytwo** [comment]
Reviewed-by:

**Comment by songbaisen** [comment]
en, lgtm.

File: src/client/Client.cc [hunk 1]
```diff
@@ -11061,7 +11061,7 @@ int Client::ll_get_stripe_osd(Inode *in, uint64_t blockno,
       pg_t pg = (pg_t)olayout.ol_pgid;
       vector<int> osds;
       int primary;
-      o.pg_to_osds(pg, &osds, &primary);
+      o.pg_to_up_acting_osds(pg, NULL, NULL, &osds, &primary);
```
Comment (xiexingguo) [inline src/client/Client.cc]: Why don't use `pg_to_acting_osds` directly here? Since the up set is irrelevant here.

Comment (liewegas) [inline src/client/Client.cc]: updated, althouh it is just a wrapper around pg_to_up_acting_osds.  also 
fixed to return primary instead of osds[0]

**Comment by xiexingguo** [comment]
lgtm also

**Pull request merged**